frontend/src/components/course-search/filter-bubble/StatusBubble.tsx

```diff
@@ -0,0 +1,36 @@
+import type * as Search from "@lib/search";
+import type { FilterBubbleComponentProps } from "./FilterBubble";
+import * as APIv4 from "hyperschedule-shared/api/v4";
+import AutoComplete from "./AutoComplete";
+import { memo } from "react";
+
+const StringToSectionStatusMap: Record<string, APIv4.SectionStatus> = {
```

Comment:
My thought was that it's an option in the data, so maybe someone would want to use it. Thinking about it, it seems that that use case would be pretty obscure and it might confuse some users so I think I agree that it should be removed.